index.bs

```diff
@@ -246,7 +246,7 @@ A [=/screen=] has an <dfn>orientation</dfn>, which is described in [[screen-orie
 
 A [=/screen=] has a <dfn>label</dfn>, which is a string that meaningfully describes the screen to a user.
 
-Note: The [=screen/label=] can be an arbitrary string selected by the user agent. It could describe the screen relative to the device, e.g. `"internal"` vs. `"external"`, it could include the dimensions, e.g. `"640×480"`, it could include hardware model information, e.g. `"Acme Telletube 1000x"`, it could include a distinguishing number, e.g. `"screen 1"` vs. `"screen 2"`, or all of the preceding.
+Note: The [=screen/label=] can be an arbitrary string selected by the user agent. It could describe the screen relative to the device, e.g. `"internal"` vs. `"external"`, it could include the dimensions, e.g. `"640×480"`, it could include hardware model information, e.g. `"Acme Telletube 1000x"`, it could include a distinguishing number, e.g. `"screen 1"` vs. `"screen 2"`, or all of the preceding. The user agent may produce an empty string if the display details are unknown.
```

Comment:
Consider: `"The label can be an empty string if underlying display details are unknown or if user agents choose to hide that information."`

Perhaps we should modify the "hardware model information" bit above to say something like:
`it could include a product name, e.g. `"Acme Telletube 1000x"` from a Display Product Name descriptor (E-EDID A.2; Section 3.10.3.4).`

I'm not sure there's a good way to link to the VESA PDF, I had to fill out a form to get this tenuous link:
https://app.box.com/s/vcocw3z73ta09txiskj7cnk6289j356b/file/769082176623
We could maybe link to wikipedia's definition: https://en.wikipedia.org/wiki/Extended_Display_Identification_Data#Display_Descriptors

Reply:
Update and added a link to wikipedia.